test/e2e/create_cluster_test.go

```diff
@@ -1452,3 +1452,72 @@ func testSwitchFromPublicToPrivate(ctx context.Context, client crclient.Client,
 		e2eutil.ValidatePrivateCluster(t, ctx, client, hostedCluster, clusterOpts)
 	}
 }
+
+func TestCreateClusterWithImageConfiguration(t *testing.T) {
```

Comment:
can we please reuse the existing func TestCreateClusterCustomConfig for this validation?

Reply:
I checked it out and the reason I didn't use it is because it's specific to AWS, and I think this test should cover both Azure and AWS. What do you think?

Reply:
Let's avoid creating an additional HC for this test. Let's adapt TestCreateClusterCustomConfigV2 to only run the current validation in aws and this one everywhere. Or if we only run this one in aws that's an acceptable start.